config.cc

```diff
@@ -52,6 +53,12 @@ namespace
       result = QDir::fromNativeSeparators( QString::fromWCharArray( _wgetenv( L"APPDATA" ) ) );
     #else
       char const * pathInHome = ".goldendict";
+      #if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
+        if (!result.exists(pathInHome)) {
+          result = QDir(QStandardPaths::locate(QStandardPaths::ConfigLocation, QString(), QStandardPaths::LocateDirectory));
```

Comment:
I think [`writableLocation()`](https://doc.qt.io/qt-5/qstandardpaths.html#writableLocation) should be used instead of `locate()`.

Reply:
You are right.